Patrycja Brzezińska.tex

```diff
@@ -0,0 +1,59 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tikz}
+\usepackage{phonrule}
+\usepackage{tikz-qtree}
+\usepackage{tipa}
+\usepackage{amsmath}
+
+\title{LaTeX assignment}
+\author{Patrycja Brzezińska}
+\date{May 2019}
+\usepackage[style=authoryear]{biblatex}
+\addbibresource{Moja biblioteka.bib}
+
+\begin{document}
+\maketitle
+\section{Section 1}
+My favorite topics in linguistics:
+\begin{itemize}
+  \item Syntax
+  \item Pragmatics
+  \item Semantics
+  \item Phonology
+  \item Morphology
+\end{itemize}
+
+\section{Section 2}
+\begin{IPA}
+/ maI neIm  Iz p\textturnv tr\textbari \texttslig j\textturnv \ \ae nd aI w\textturnscripta z bO:n In tt\:sev | wI\textteshlig \ Iz \textschwa \ sIti nI\textschwa \ D\textschwa \ nO:T E\textdyoghlig \ \textturnscripta v ˈp\textschwa Ul\textschwa nd || It leIz saUT \textturnscripta v gda\textltailn sk, we\textschwa r aI h\ae v biːn ˈst\textturnv diIN ˈn\ae \textteshlig r\textschwa l ˈl\ae NgwI:\textdyoghlig \ ˈpr\textschwa UsEsIN || mOjE hObbi tO t\:s\textbari ta\textltailn E kC\~O\:zek | \:st\:sEgul\textltailn E mwO\textdctzlig E\:zOv\textbari G lub fantast\textbari ki || v t\:saCE vOln\textbari m sp\~E:\textdzlig am t\:sas z GwOpak\super jEm lub p\:z\textbari ja\texttctclig uwm\super ji zE stud\super juv
```

Comment:
1. \textturn in your name? just /a/ would be much better
2. Where are stress marks?
3. please add weak forms for words like and, of...
4. no comma in transcription :)
5. why length mark in language?
6. please correct Polish cz :)
7. młodzieżowych - [x] at the end, why voiced [G]? the same for <ch>łopakiem
8. przyjaciółmi - /p/ devoices the following fricative
9. studiów - final devoicing